lib/src/modules/task.cc

```diff
@@ -150,7 +150,7 @@ Task::Task(const fs::path& exec_prefix,
     task_download_connect_timeout_ { task_download_connect_timeout },
     task_download_timeout_ { task_download_timeout },
 #ifdef _WIN32
-    features_ { "puppet-agent", "powershell" }
+    features_ { "puppet-agent", "powershell" },
```

Comment:
This change broke the appveyor build:

the #ifdef clause here is switching out how features_ is set for windows vs. Unix. In the case of Windows builds this will be the last line rather than part of a list.

Reply:
Thanks, yeah that was an artifact left over from testing what happens if the mutex was managed as an instance variable on individual module objects. Will clean up.